### PR TITLE
Ep 178

### PR DESCRIPTION
**Pull request by a-zakirov**
Changed source URL for dragonboard410c_bootloader_emmc_linux-88.zip

### CODE REVIEW

**Comment by balazs-gollner-epam** [comment]
Please also merge it to sumo branch as that's the one used in the medianode-distro repo as git submodule

File: recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb [hunk 1]
```diff
@@ -2,7 +2,7 @@ DESCRIPTION = "Qualcomm primary bootloader binary blobs for Dragonboard"
 LICENSE = "Proprietary"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=4d087ee0965cb059f1b2f9429e166f64"
 
-SRC_URI = "http://builds.96boards.org/releases/dragonboard410c/linaro/rescue/${PV}/dragonboard410c_bootloader_emmc_linux-88.zip \
+SRC_URI = "https://releases.linaro.org/96boards/dragonboard410c/linaro/rescue/17.09/dragonboard410c_bootloader_emmc_linux-88.zip \
```
Comment (balazs-gollner-epam) [inline recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb]: Is there a specific reason to not use ${PV}

File: recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb [hunk 2]
```diff
@@ -25,7 +25,7 @@ do_compile() {
 }
 do_install() {
     install -d ${D}${libdir}/${PN}
-    for f in NON-HLOS.bin hyp.mbn rpm.mbn tz.mbn sbl1.mbn LICENSE; do
+    for f in sbc_1.0_8016.bin hyp.mbn rpm.mbn tz.mbn sbl1.mbn LICENSE; do
```
Comment (balazs-gollner-epam) [inline recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb]: What is this change?

Comment (a-zakirov) [reply recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb]: This is Matt Madison fix to comply real archive contents.
bootloader-emmc-linux: fix installed files list

File: recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb (hunk 1 above)
Comment (a-zakirov) [reply recipes-bsp/bootloader/bootloader-emmc-linux_17.09.bb]: Fixed in d0fda0c..069451d

**Pull request merged**